server/src/db/migrations/20220131193845_addCharacters.cjs

```diff
@@ -0,0 +1,26 @@
+/**
+ * @typedef {import("knex")} Knex
+ */
+
+/**
+ * @param {Knex} knex
+ */
+exports.up = async (knex) => {
+  return knex.schema.createTable("characters", (table) => {
+    table.bigIncrements("id")
+    table.string("name").notNullable()
+    table.string("gameTitle")
+    table.string("gameSeries").notNullable()
+    table.string("pictureUrl")
```

Comment:
If we are adding a property in our migration we should use it in some way, especially if we are seeding data as well.
You should either remove `picutreUrl` or seed data at least in some of the records

Reply:
Images were part of our plan from the beginning; they are used in the show page which has been added locally and is waiting for this merge to be completed before itself being PR'd